packages/react-tabster/src/hooks/useKeyborg.ts

```diff
@@ -0,0 +1,38 @@
+import { createKeyborg } from 'keyborg';
+import { KeyborgCallback } from 'keyborg/dist/Keyborg';
+import { RefObject, useEffect, useMemo, useRef } from 'react';
+import { KEYBOARD_NAV_ATTRIBUTE } from '../symbols';
+import { useConst } from '@fluentui/react-utilities';
+import { useFluent } from '@fluentui/react-shared-contexts';
+
+/**
+ * instantiates keyborg and add attribute to ensure focus indicator synced to keyborg logic
+ */
+export function useKeyborg<E extends HTMLElement>() {
```

Comment:
I don't wanna be that guy but, we are making a layer over `tabster` and our partners like more boring enterprise names. Should we rename this to like `useKeyboardNavigationIndicator` ?

Reply:
![image](https://user-images.githubusercontent.com/5483269/119795536-daeb5000-bed8-11eb-81bf-cf19a22e7bb1.png)

Reply:
memes apart, you're completely right! how about `useKeyboardNavAttribute`? @ling1726

Reply:
yup, sounds good